### ISSUE TITLE
[BUG] Progress Bar is missing Aria Label

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#506]
https://modus-web-components.trimble.com/?path=/docs/components-progress-bar--default

![image](https://user-images.githubusercontent.com/1212885/175456916-82050aa2-a24b-46a0-b47e-cf5d1905e2aa.png)

See https://modus-bootstrap.trimble.com/components/progress-bars/ for implementation details.
